README.md

```diff
@@ -1 +1,10 @@
-# hello-world
\ No newline at end of file
+# hello-world
+
+Top 5 Albums
+
+1. Frank Ocean - Blonde
+2. Drake - More Life
+2. Travis Scott - Astro World
+4. Kanye West - My Beautiful Dark Twisted Fantasy 
```

Comment:
I'm not a huge fan of Kanye West these days, but I do love this album. Good choice.

Reply:
This is good to go!